### DISCUSSION TITLE
Missing Content Icon

### DISCUSSION THREAD

**Discussion by cdanny358** [directus/directus#13592] [Archive]
Ever since I upgraded Directus to v9 from one of the beta versions of v9, the main content icon in the left column has been missing. I can get to the content just fine by typing in the proper URL, but there's no other way to get there without the icon. Has anyone else seen this issue? Is there a fix?
![Missing Icon](https://user-images.githubusercontent.com/390911/170731500-d759cff8-b2ed-4f5a-bf58-d6b130973d4b.png)

**Comment by rijkvanzanten** [comment]
Do you have it disabled in Project Settings perhaps?

**Reply by cdanny358** [reply]
Well, I'll be hornswaggled. I had no idea that was a possibility, but yes, it's disabled. I think somehow it toggled when 'collections' became 'content' some months ago. Thanks!
